euth/communitydebate/templates/euth_communitydebate/topic_form.html

```diff
@@ -1,7 +1,14 @@
 {% extends "base.html" %}
-{% load widget_tweaks i18n thumbnail base_tags form_tags %}
+{% load static widget_tweaks i18n thumbnail base_tags form_tags %}
+
+{% block title %}
+{% if mode == 'create' %}
+{% trans 'Create a new topic' %}
+{% elif mode == 'update' %}
```

Comment:
Good idea! One could do with {% else %}, though, so that there is always something there (and it's one check less).